packages/web/src/pages/deactivate-account-page/store/sagas.ts

```diff
@@ -22,7 +22,7 @@ const DEACTIVATE_CONFIRMATION_UID = 'DEACTIVATE'
 function* handleDeactivateAccount() {
   try {
     yield call(waitForBackendSetup)
-    const accountUserId: Nullable<ID> = yield select(getUserId)
+    const accountUserId: ID = yield select(getUserId)
```

Comment:
can we yield* these or would that take too long? from what i remember this actually is a `Nullable<ID>`, so maybe we just return if missing or smthg?

Reply:
yeah just tried it and after changing 'redux-saga/effects' import to `typed-redux-saga/macro`, this logic works: 

```typescript
    const accountUserId = yield* select(getUserId)
    const userMetadata = yield* select(getAccountUser)
    if (!accountUserId || !userMetadata) return

    yield put(
      requestConfirmation(
        DEACTIVATE_CONFIRMATION_UID,
...
```

Reply:
Sick ok will do that instead